wiki/Tournaments/UKCC/X/en.md

```diff
@@ -6,13 +6,610 @@ tags:
+- In order to reschedule, you must contact your opponents, agree on a time, and be able to provide **proof** (by form of a screenshot/image) of the agreed reschedule to the hosts.
+- In order to ensure that as many matches take place as possible, reschedules are allowed up to **2 hours before your match is due to be played**. However, your opponent holds the right to **refuse a reschedule if asked less than 12 hours before the scheduled time**.
+- If it is impossible to find a time to reschedule to that suits both teams, both teams should talk to the hosts (![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) or ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973)) who will attempt to sort something out.
+
+### Pre-match
+
+- Referees will create the lobby approximately **10 minutes** before the match is due to start.
+- Players have a 10-minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will receive losses.
+- We take no-shows seriously. If you fail to show up to your match without valid reason, you may forfeit your right to participate in future UKCC tournaments.
+- The referee cannot play in the match.
+
+### Qualifiers
+
+- Players will play each map in the Qualifiers mappool once.
+- There will be 3 NoMod, 2 Hidden, 2 Hard Rock and 2 Double Time maps.
+- The top 32 scores will progress into the Swiss bracket.
+
+### Swiss
+
+- Since Swiss bracketing is a natural seeding system, matchups will be drawn between **top 16 and bottom 16 players randomly** to begin.
+- Two matches will happen every weekend for each player, one on Saturday and one on Sunday - therefore, **two matchups will be drawn per weekend** for each player.
+- The order the matches are played in **does not matter**.
+- Matchups will be drawn between players with the **same aggregate match W-L-D score** (eg. players at 3-0-1 will prioritise playing against the players with the most similar W-L-D), and will prioritise **avoiding rematches** until each active player has played each other once.
+
+### Rolling
+
+- When both players are in the lobby and ready, the referee will ask the players to `!roll`. The order of picks of bans is decided using the [UKCC 4 roll system](https://i.imgur.com/S0VIvni.png).
+
+### Banning
+
+- Each player will have **one** ban.
+- Players have **2 minutes** to ban a map. Failure to ban a map within this time limit results in a lost ban.
+- There are no restrictions on bans and picks. Players are free to ban/pick any map regardless of mod.
+
+### Warmups
+
+- There will be **no** warmups played.
+
+### Picking
+
+- Players have **90 seconds** to pick a map. Failure to pick a map within this time limit results in pick choice being passed to the opposing player.
+- **Failed scores will count, and every map will be played with the NF modification enabled**
```

Comment:
it would be "modifier" and not "modification" anyway

```suggestion
- **Failed scores will count, and every map will be played with the No Fail mod enabled**
```